src/nautilus_librarian/mods/libvips/domain/validate_image_dimensions.py

```diff
@@ -0,0 +1,26 @@
+from nautilus_librarian.mods.libvips.domain.utils import get_image_dimensions
+
+
+class file_dimensions_exception(ValueError):
```

Comment:
For class names, we are using PascalCase.

Reply:
I would call it `InvalidImageDimensions` or something like that. I'm not using any suffix like `InvalidMediaFileDimensionsException`.